schema/gene.js

```diff
@@ -27,119 +35,145 @@ export const GeneType = new GraphQLObjectType({
   name: "Gene",
   interfaces: [NodeInterface],
   isTypeOf: obj => _.has(obj, "browseable") && (_.has(obj, "published") || _.has(obj, "family")),
-  fields: {
-    ...GravityIDFields,
-    cached,
-    artists: {
-      type: new GraphQLList(Artist.type),
-      resolve: ({ id }) => {
-        return gravity(`gene/${id}/artists`, {
-          exclude_artists_without_artworks: true,
-        })
+  fields: () => {
```

Comment:
This is a work around for the `geneConnection` variable being circulated. It's a bit scary to put all the field definitions into a function call, but we weren't sure how we could avoid the curricular reference.

Reply:
👍  using a thunk is best practices for this type of situation!

Reply:
Aye 👍 

Also, thunk thunk thunk thunk, what a name

Reply:
What is a thunk?

Reply:
Good Q. I do not know either @alloy pointed me to this - https://github.com/artsy/metaphysics/pull/811#discussion_r151270893

Reply:
In short, it’s a way to get a value lazily, that is only when you need it. https://en.wikipedia.org/wiki/Thunk